src/actions/index.js

```diff
@@ -1,3 +1,4 @@
 export * from './UserActions';
 export * from './ProvidersActions';
 export * from './ProductsActions';
+export * from './EventsActions';
```

Comment:
```suggestion
export * from './EventsActions';

```
Please add EOF line

Reply:
I see it on my code editor 

![image](https://user-images.githubusercontent.com/66505715/173159158-b744ee46-4266-4f07-8ba5-091f2c4e74b0.png)

I don't know why GitHub is not showing it :( But I think GitHub show an EOF error when it exists

Reply:
I remember there were 2 settings, one in git configuration (somethink like "clrf") and other in vscode (like addEndOfLine sorry I am from phone). Please check if you can add those on this project.